### DISCUSSION TITLE
e2e

### DISCUSSION THREAD

**Discussion by coolemur** [trpc/trpc#2479] [General]
Hi,

Is it just me, or e2e 'server-side rendering test' is failing in starter project ?

Using initial setup with no additional changes:
[https://trpc.io/docs/example-apps](https://trpc.io/docs/example-apps) -> "Quick start with create-next-app"

App starts, first 3 tests passes and the last one fails.

Any ideas why ?

**Comment by KATT** [comment]
It doesn't fail here https://github.com/trpc/examples-next-prisma-starter

**Reply by coolemur** [reply]
Good point.

Just checked that in Github actions it shows test as passed.
But when I try to run the test on localhost:3000, last test (4) fails. Something to do with ssr.


<img width="950" alt="Screenshot 2022-08-21 at 22 25 23" src="https://user-images.githubusercontent.com/29693103/185809614-2450be72-8ef5-41f4-9866-8d9b4128baa1.png">
